src/Microsoft.ServiceFabric.Actors/Diagnostics/PerformanceCounters/ActorLockContentionCounterWriter.cs

```diff
@@ -3,22 +3,24 @@
 // Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
 // ------------------------------------------------------------
 
-namespace Microsoft.ServiceFabric.Actors.Diagnostics
+namespace Microsoft.ServiceFabric.Actors.Diagnostics.PerformanceCounters
 {
     using System.Fabric.Common;
 
     // This class modifies the value of the performance counter that represents the
     // number of pending actor calls that are waiting for the actor lock.
     internal class ActorLockContentionCounterWriter : FabricBaselessPerformanceCounterWriter
     {
+        protected ActorLockContentionCounterWriter() { }
+
         internal ActorLockContentionCounterWriter(FabricPerformanceCounterSetInstance counterSetInstance)
             : base(
                 counterSetInstance,
                 ActorPerformanceCounters.ActorCallsWaitingForLockCounterName)
         {
         }
 
-        internal void UpdateCounterValue(PendingActorMethodDiagnosticData pendingMethodData)
+        internal virtual void UpdateCounterValue(PendingActorMethodDiagnosticData pendingMethodData)
         {
```

Comment:
Forgot to mention last time. When I was thinking about how to unit test this code, I though we could use Inspector to get to the FabricPerformanceCounter obtained by the base class and use its GetValue method. This is not as invasive as making the UpdateCounterValue methods virtual and would be closer to the state-based testing, which is generally more maintainable than behavior-based testing we do with mocks. It's a pretty wide gray area though, so just a suggestion to consider.